website/docs/guides/open-telemetry.md

```diff
@@ -18,6 +25,13 @@ services.AddKafka(
 );
 ```
 
-## Using KafkaFlow instrumentation with .NET Automatic Instrumentation
+Once you have your .NET application instrumentation configured ([see here](https://opentelemetry.io/docs/instrumentation/net/getting-started/)), you just need to subscribe to the source `KafkaFlow.OpenTelemetry` that is accessible through a constant at `KafkaFlowInstrumentation.ActivitySourceName`.
+
+## Using .NET Automatic Instrumentation
 
 When using [.NET automatic instrumentation](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation), the KafkaFlow activity can be captured by including the ActivitySource name `KafkaFlow.OpenTelemetry` as a parameter to the variable `OTEL_DOTNET_AUTO_TRACES_ADDITIONAL_SOURCES`.
+
+## Propagation
+
+KafkaFlow supports [Propagation](https://opentelemetry.io/docs/specs/otel/context/api-propagators/), the mechanism that moves context information data between services and processes.
```

Comment:
I wouldn't say that we _support_ Propagation, as it is not something that the end user will use. Maybe we can change to `KafkaFlow OTEL is built on [Propagation]` or `KafkaFlow uses [Propagation]`.

Reply:
👍 updated.